### PR TITLE
Creator DAO article

### PR DESCRIPTION
**Pull request by gordoncc**

### CODE REVIEW

File: content/blog/urbit-creator-daos.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<div style="font-size:13px; text-align: center; color: #999">Murphy already offers his members access to the Other Life urbit group.</div>
+
+Finally, Murphy cites Urbit’s durability.
+
+> The second most important Urbit trait after ownership is durability. The creator economy tools change all the time. Every time you try to move your people from one new app to another app, you lose a lot, right? You want to be an influencer, you want to be an important thinker and speaker. All of a sudden, you’ve got to be on Clubhouse! This means telling the people on your newsletter, telling your Patreon subscribers, telling the people on your podcast, hey, download this cool new app called Clubhouse and find me there. It's going to be awesome. It's so cringe and corny and only a small number of them will even make it over there. Then all of a sudden, no one's on Clubhouse and no one wants to use Clubhouse anymore. This is a major problem for people navigating the creator economy. I think one of the big selling points of Urbit is—this is the one place that I’ll always be. I’ve been working on my Urbit for over 10 years. This thing is going to be the nuclear bunker of the internet.
```
Comment (thelifeandtimes) [inline content/blog/urbit-creator-daos.md:68]: 97% sure jmurphy hasn't been on urbit for a decade. I think I understand the context here as 'in the future he would be able to say something like this' but it's not totally clear if you don't know the broader context of justin on urbit.

File: content/blog/urbit-creator-daos.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+author = "Isaac Simpson"
+ship = "~mogmet-tadnem"
+image = "https://storage.googleapis.com/media.urbit.org/blog/creatordao-3.jpg"
++++
+
+![Image](https://storage.googleapis.com/media.urbit.org/blog/creatordao-3.jpg)
+<div style="font-size:13px; text-align: center; color: #999">Photo by <a href="https://unsplash.com/@nathynmasters
+">Nathyn Masters</a></div>
+
+# Otherlife: Proto Creator DAO
```
Comment (thelifeandtimes) [inline content/blog/urbit-creator-daos.md]: does `Otherlife` need a space or any capitalization?

File: content/blog/urbit-creator-daos.md (hunk 1 above)
Comment (thelifeandtimes) [inline content/blog/urbit-creator-daos.md]: 97% sure jmurphy hasn't been on urbit for a decade. I think I understand the context here as 'in the future he would be able to say something like this' but it's not totally clear if you don't know the broader context of justin on urbit.

**Review by thelifeandtimes — APPROVED** [review]
if my comments are overruled, happy for this to be merged.

File: content/blog/urbit-creator-daos.md (hunk 2 above)
Comment (gordoncc) [reply content/blog/urbit-creator-daos.md]: Yes, good catch

File: content/blog/urbit-creator-daos.md (hunk 1 above)
Comment (gordoncc) [reply content/blog/urbit-creator-daos.md]: Right, I'll clarify

**Pull request merged**